mod/poco.php

```diff
@@ -27,6 +27,12 @@ function poco_init(App $a) {
 	$justme = false;
 	$global = false;
 
+	if($a->argc > 1 && $a->argv[1] === '@server') {
```

Comment:
Please add a docu line what happens if the condition is fulfilled (if you know it for the conditions below it would be nice to also add a some docu)